src/main/java/com/google/maps/model/AutocompletePrediction.java

```diff
@@ -59,9 +59,19 @@
     public int offset;
   }
 
+  /**
+   * Describe the location of the entered term in the prediction result text, so that the term can
+   * be highlighted if desired.
+   */
   public MatchedSubstring matchedSubstrings[];
 
   /**
+   * Structured formatting contains a description of how the autocomplete query matched the returned
```

Comment:
I'd simplify to `A description of how the autocomplete query matched the returned result.`. Leading with "Structured formatting contains..." is basically repeating the class name.